src/javascript/binary/base/language.js

```diff
@@ -48,7 +49,7 @@ const Language = (() => {
             const crowdin_lang = Cookies.get('jipt_language_code_binary-static'); // selected language for in-context translation
             if (crowdin_lang) {
                 current_lang = crowdin_lang.toUpperCase().replace('-', '_').toUpperCase();
-                $('body').addClass(current_lang); // set the body class removed by crowdin code
+                document.getElementsByTagName('body')[0].addClass(current_lang); // set the body class removed by crowdin code
```

Comment:
shouldn't this be `document.getElementsByTagName('body')[0].classList.add(current_lang)` ?

Reply:
yes, thank you

Reply:
I suspect there are more cases I've missed, so would appreciate it if you could review with wide eyes :)